hcipy/optics/fiber.py

```diff
@@ -64,6 +66,17 @@ def core_radius(self, core_radius):
 		self._core_radius = core_radius
 		self.clear_cache()
 
+	@property
+	def position(self):
```

Comment:
Why mix `position` and `center`? I'd rather choose one or the other. My preference goes to `position`, but feel free to let me know any reason for the other name.

Reply:
I have adjusted the name. There was no particular reason why they were different.